renormalizer/mps/tests/test_evolve.py

```diff
@@ -71,7 +71,9 @@ def test_tdvp_vmf(init_state, with_mu, force_ovlp, atol):
 def test_tdvp_cmf(init_state):
     mps = init_state.copy()
     mps.evolve_config  = EvolveConfig(EvolveMethod.tdvp_mu_cmf)
-    check_result(mps, mpo, 0.01, 0.5, 5e-4)
+    for flag in [True, False]:
```

Comment:
It'll be more consistent if the option is moved to the decorator `pytest.mark.parametrize`

Reply:
fixed.